app/src/main/java/com/esri/arcgisruntime/opensourceapps/datacollection/views/popup/PopupView.kt

```diff
@@ -44,9 +60,36 @@ class PopupView : FrameLayout {
     var popup: Popup? = null
```

Comment:
Should `popup` be nullable? I assume a PopupView always has to be associated with a non-null `Popup`

Reply:
I had a custom setter for popup which required popup to be initialized. This logic is no longer in place.